integration-tests/tests/workspace-dashboard.js

```diff
@@ -41,23 +63,24 @@ const workspaceDashboardPage = (testPage, testUrl, token, workspaceName) => {
 
     assertLockWorkspace: async () => {
       await assertTextNotFound(testPage, 'Workspace is locked')
-      await click(testPage, clickable({ text: 'Workspace Action Menu' }))
+      await clickWorkspaceActionMenu(testPage)
       await click(testPage, clickable({ textContains: 'Lock' }))
-      await noSpinnersAfter(testPage, { action: () => click(page, clickable({ textContains: 'Lock Workspace' })) })
+      await noSpinnersAfter(testPage, { action: () => click(testPage, clickable({ text: 'Lock Workspace' })) })
       await findText(testPage, 'Workspace is locked')
     },
 
     assertUnlockWorkspace: async () => {
+      await dismissNotifications(testPage)
       await findText(testPage, 'Workspace is locked')
-      await click(testPage, clickable({ text: 'Workspace Action Menu' }))
+      await clickWorkspaceActionMenu(testPage)
       await click(testPage, clickable({ textContains: 'Unlock' }))
-      await noSpinnersAfter(testPage, { action: () => click(page, clickable({ textContains: 'Unlock Workspace' })) })
+      await noSpinnersAfter(testPage, { action: () => click(testPage, clickable({ text: 'Unlock Workspace' })) })
       await assertTextNotFound(testPage, 'Workspace is locked')
     },
 
     assertTabs: async (expectedTabs, enabled) => {
       await Promise.all(_.map(async tab => {
-        await (enabled ? clickNavChildAndLoad(testPage, tab) : assertNavChildNotFound(testPage, tab))
+        await (enabled ? testPage.waitForXPath(navChild(tab)) : assertNavChildNotFound(testPage, tab))
```

Comment:
Clicking on the tabs leads to some flakiness, and really I just care whether the tabs exist.